commands/release.go

```diff
@@ -424,6 +433,37 @@ func editRelease(cmd *Command, args *Args) {
 	args.NoForward()
 }
 
+func deleteRelease(cmd *Command, args *Args) {
+	tagName := cmd.Arg(0)
+	if tagName == "" {
+		utils.Check(fmt.Errorf("Missing argument TAG"))
+		return
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.CurrentProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	release, err := gh.FetchRelease(project, tagName)
+	utils.Check(err)
+
+	/*cs := git.CommentChar()
+	message, err := renderReleaseTpl("Deleting", cs, tagName, project.String(), flagReleaseCommitish)
+	utils.Check(err)
+	message = fmt.Sprintf("%s\n\n%s\n%s", release.Name, release.Body, message)*/
+	message := fmt.Sprintf("Deleting release related to %s...", tagName)
```

Comment:
A command shouldn't output what it's doing, only if there's something useful as a result (such as the URL of a freshly created release). This command doesn't have anything useful to report, so it shouldn't print anything.

However, hub commands can be run in `--noop` mode, where they simply print what would be done, but don't actually make the changes. In this mode, it's fine to read from the API, but not to write to it. Instead, it should output something like `Would delete release for tag %s`.